src/main/java/org/generationcp/breeding/manager/listmanager/GermplasmSearchResultsComponent.java

```diff
@@ -260,11 +261,20 @@ public String generateDescription(final Component source, final Object itemId, f
 				// set the default value to empty string instead of null for germplasm without name
 				String germplasmNames = "";
 				if (propertyId == GermplasmSearchResultsComponent.NAMES) {
-					final Item item = GermplasmSearchResultsComponent.this.matchingGermplasmTable.getItem(itemId);
-					final Integer gid =
-							Integer.valueOf(((Button) item.getItemProperty(ColumnLabels.GID.getName()).getValue()).getCaption());
+					try{
+						final Item item = GermplasmSearchResultsComponent.this.matchingGermplasmTable.getItem(itemId);
+						final int gid =
+								Integer.parseInt(((Button) item.getItemProperty(ColumnLabels.GID.getName()).getValue()).getCaption());
+
+						germplasmNames = GermplasmSearchResultsComponent.this.getGermplasmNames(gid);
+					}catch(MiddlewareException ex){
+						if(ex.getMessage().contains("Problem building pedigree string for gid")){
```

Comment:
Maybe extract this as constant in PedigreeServiceImpl: `Problem building pedigree string for gid` to be referenced here

Reply:
You're swallowing the exception here for the case of non-cross expansion related exception. 
2 options:
1. Re-throw the exception
2. Display error message containing the exception's message